spec/001-initialization.js

```diff
@@ -329,6 +329,66 @@ describe('ConstructorioID', function () {
       var session = new ConstructorioID();
       expect(session.on_node).to.be.false;
     });
+
+    describe('when used with different libraries on different subdomains', function () {
+      var dom;
+
+      beforeEach(function () {
+        dom = new jsdom.JSDOM('', {
+          url: 'http://checkout.localhost.com'
+        });
+        global.window = dom.window;
+        global.window.localStorage = helper.getStorageMock();
+        global.document = dom.window.document;
+      });
+
+      afterEach(function () {
+        delete global.window;
+        delete global.document;
+      });
+
+      it('should move client_id to the top level domain', function () {
```

Comment:
Should this say something like `should set client_id on the specified cookie_domain` since the module doesn't specifically move it to the top level domain on it's own?

Reply:
True that. Updated